go/teams/service_helper.go

```diff
@@ -61,6 +61,7 @@ func Details(ctx context.Context, g *libkb.GlobalContext, name string) (res keyb
 	res.AnnotatedActiveInvites = annotatedInvites
 
 	membersHideInactiveDuplicates(ctx, g, &res.Members)
+	membersHideDeletedUsers(ctx, g, &res.Members)
```

Comment:
I'm not sure how much this matters but maybe these two above can be collapsed into one, then you'd be iterating over all the lists only once (not counting rest of the code which does this a few times probably)